.github/workflows/pr-rebase-needed.yml

```diff
@@ -0,0 +1,20 @@
+name: "PR Needs Rebase"
+on:
+  # So that PRs touching the same files as the push are updated
+  push:
+  # So that the `dirtyLabel` is removed if conflicts are resolve
+  # We recommend `pull_request_target` so that github secrets are available.
+  # In `pull_request` we wouldn't be able to change labels of fork PRs
+  pull_request_target:
+    types: [synchronize]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check for merge conflicts
+        uses: eps1lon/actions-label-merge-conflict@releases/2.x
+        with:
+          dirtyLabel: "rebase needed :construction:"
+          repoToken: "${{ secrets.GITHUB_TOKEN }}"
+          commentOnDirty: "This pull request has merge conflicts that must be resolved before we can merge this."
```

Comment:
Can you test if this comment can be posted more than once on a single PR? That would be annoying.

Reply:
I did test this and found the comment is only applied once when merge conflicts are first detected!